apps/maestro/src/features/RegisterRemoteTokens/hooks/useRegisterRemoteCanonicalTokens.ts

```diff
@@ -56,13 +56,16 @@ export default function useRegisterRemoteCanonicalTokens(
     return destinationChainIds.map((axelarChainId, i) => {
       const gasValue = gasFees[i];
 
-      return encodeInterchainTokenServiceDeployRemoteCanonicalTokenData({
-        tokenId: tokenDetails.tokenId,
-        destinationChain: axelarChainId,
-        gasValue,
-      });
+      return INTERCHAIN_TOKEN_FACTORY_ENCODERS.deployRemoteCanonicalInterchainToken.data(
+        {
+          originalChain: sourceChain?.id ?? "0x",
```

Comment:
we now allow deploying remote tokens from non origin chains as well. the token just needs to have been deployed on that chain first